File: src/c_cpp/quality_demo.c
#include <stdio.h>

int main(void) {
    double amounts[] = {125.5, -25.0, 900.0, 42.0};
    int n = 4;
    double sum = 0.0;
    int negative = 0;
    for (int i = 0; i < n; i++) {
        sum += amounts[i];
        if (amounts[i] < 0) negative++;
    }
    printf("average=%.2f negative_amounts=%d\n", sum / n, negative);
    return 0;
}

File: src/csharp/QualityDemo.cs
using System;
using System.Linq;

public class QualityDemo
{
    public static void Main()
    {
        double[] amounts = {125.5, -25.0, 900.0, 42.0};
        Console.WriteLine($"average={amounts.Average():F2} negative_amounts={amounts.Count(x => x < 0)}");
    }
}
